server/dialect-idms/src/main/antlr4/org/eclipse/lsp/cobol/dialects/idms/IdmsParser.g4

```diff
@@ -401,7 +401,7 @@ findObtainClause
     ;
 
 calcClause
-    : (CALC | ANY | DUPLICATE) idms_db_entity_name
+    : (CALC | ANY | DUPLICATE) idms_db_entity_name idmsOnClause?
```

Comment:
Should the `idmsOnClause` be attached to `findObtainClause` instead? The following seems valid:
```
OBTAIN CURRENT EMPLOYEE
    ON DB-REC-NOT-FOUND
```

Reply:
Good point